salt/client/ssh/wrapper/state.py

```diff
@@ -32,6 +32,60 @@
 log = logging.getLogger(__name__)
 
 
+def _ssh_state(chunks, __opts__, __context__, __pillar__, __salt__, st_kwargs,
+              kwargs, test=False):
+    '''
+    funciton to run a state with the given chunk via salt-ssh
+    '''
+    file_refs = salt.client.ssh.state.lowstate_file_refs(
+            chunks,
+            _merge_extra_filerefs(
+                kwargs.get('extra_filerefs', ''),
+                __opts__.get('extra_filerefs', '')
+                )
+            )
+    # Create the tar containing the state pkg and relevant files.
+    trans_tar = salt.client.ssh.state.prep_trans_tar(
+            __opts__,
```

Comment:
@Ch3LL  This is unused inside the function. I'd propose either remove the option entirely from `prep_trans_tar` or refactor that it is used.